plugins/fabric8-multi-tenant-manager/src/main/java/com/redhat/che/multitenant/RhCheInfraEnvironmentProvisioner.java

```diff
@@ -64,7 +64,7 @@ public RhCheInfraEnvironmentProvisioner(
       @Named("che.infra.kubernetes.pvc.enabled") boolean pvcEnabled,
       OpenShiftUniqueNamesProvisioner uniqueNamesProvisioner,
       RouteTlsProvisioner routeTlsProvisioner,
-      OpenShiftServersConverter openShiftServersConverter,
+      ServersConverter openShiftServersConverter,
```

Comment:
The usage of this injection should produce a warning. I think we might need to use type on injection of generic.

Reply:
Fixed in commit https://github.com/redhat-developer/rh-che/pull/646/commits/c0990d663ff5f12400a772a786b209eb4f339a02